From 262c90f269ebb9a53551bcff8cfbc8183116a6ae Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Thu, 13 Oct 2016 23:07:10 +0300
Subject: [PATCH 13/23] gas: testsuite: Mark lns-common-1-alt variant for PRU

2016-10-13  Dimitar Dimitrov <dimitar@dinux.eu>

	* gas/testsuite/gas/lns/lns.exp: Mark lns-common-1-alt variant for PRU.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 gas/testsuite/gas/lns/lns.exp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gas/testsuite/gas/lns/lns.exp b/gas/testsuite/gas/lns/lns.exp
index 81e0396..acf9947 100644
--- a/gas/testsuite/gas/lns/lns.exp
+++ b/gas/testsuite/gas/lns/lns.exp
@@ -37,6 +37,7 @@ if {
 	 || [istarget mn10*-*-*]
 	 || [istarget msp430-*-*]
 	 || [istarget nds32*-*-*]
+	 || [istarget pru-*-*]
 	 || [istarget rl78-*-*]
 	 || [istarget xtensa*-*-*] } {
       run_dump_test "lns-common-1-alt"
-- 
2.10.2

